stack-8.8.1.yaml

```diff
@@ -0,0 +1,58 @@
+resolver: nightly-2019-12-31
+packages:
+- .
+- hie-plugin-api
+
+extra-deps:
+# - ./submodules/HaRe
+
+- apply-refact-0.7.0.0
+- bytestring-trie-0.2.5.0
+- cabal-helper-1.0.0.0
+# - clock-0.7.2
+# - constrained-dynamic-0.1.0.0
+# - floskell-0.10.2
+# - fold-debounce-0.2.0.9
+# - haddock-api-2.23.0
+# - haddock-library-1.8.0
+# - haskell-src-exts-1.21.1
+# - hie-bios-0.3.2
+# # - hoogle-5.0.17.13
+# - haskell-lsp-0.19.0.0
+# - haskell-lsp-types-0.19.0.0
+# - hoogle-5.0.17.11
+# - hsimport-0.11.0
+# - lsp-test-0.10.0.0
+# - monad-dijkstra-0.1.1.2
+# - semigroups-0.18.5
+# - temporary-1.2.1.1
```

Comment:
Can you remove this (except the previously commented-out hoogle-5.0.17.13), and ensure the deps are sorted.

Reply:
oops